index.js

```diff
@@ -0,0 +1,68 @@
+// EXERCISE 1
+console.log(document.getElementById('mainTitle'));
+
+// EXERCISE 2
```

Comment:
Only text content must be modified, not html contained in element.

- [ ] Fix logic.

Reply:
Fixed.